fireplace-swing/src/main/java/io/github/bric3/fireplace/flamegraph/FlamegraphView.java

```diff
@@ -804,8 +804,11 @@ private static class FlamegraphScrollPaneMouseInputListener<T> implements MouseI
         private HoverListener<T> hoverListener;
         private FrameBox<T> hoveredFrame;
 
-        public FlamegraphScrollPaneMouseInputListener(FlamegraphCanvas<T> canvas) {
+        private boolean zoomHorizontalOnly;
```

Comment:
**chore:** I think adding getter setter here is fine as it is not breaking compatibility.